frontend/src/App.tsx

```diff
@@ -25,8 +36,15 @@ export default function App() {
     return (
         <>
             <h2>Concentration</h2>
-            {playingCards.map((playingCard: string, index) =>
-                <div key={index}>{playingCard}</div>)}
+            <div>
+                {playingCards.map((playingCard: string) =>
+                    <div key={playingCard}>{playingCard}</div>)}
+            </div>
+            <div>
+                <button onClick={loadCharacterCards}>CharacterCards</button>
+                {characterCards.map((characterCard: CharacterCard) =>
+                    <div key={characterCard.id}>{characterCard.character}</div>)}
+            </div>
```

Comment:
try to use shortcut `control + command + L` in IntelliJ to format your code from time to time. This Dom tree seems a little bit messy and hard to read.

Reply:
👍